pyvelox/test/test_expression.py

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pyvelox.pyvelox as pv
+import unittest
+
+
+class TestVeloxExpression(unittest.TestCase):
+    def test_from_string(self):
+        pv.Expression.from_string("a + b")
+
+    def test_eval(self):
+        expr = pv.Expression.from_string("a + b")
+        a = pv.from_list([1, 2, 3, None])
```

Comment:
Can you add some test expressions with : 
 1. a function
 2. expression with cast
 3. provide more arguments than required
 4. vectors with all nulls
 5. expression that works on strings, ints, doubles (to have some sort of exhaustive checks)
 6. a somewhat nested expression
 7. expression with switch/case